src/main/java/com/google/sps/servlets/AccountData.java

```diff
@@ -18,16 +18,30 @@
 /** Servlet that returns all account related info (links to login pages/account info). */
 @WebServlet("/account")
 public class AccountData extends HttpServlet {
```

Comment:
May want to consider renaming this to "AccountDataServlet" to make it clear it's an HttpServlet. As it is now, "AccountData" sounds very similar in function to the "UserInfo" class you're adding even though they're actually very different things. That would also be consistent with the other HttpServlet you have called "SuggestionServlet".

Reply:
done